tutorials/lxd/use-cuda-within-lxd/use-cuda-within-lxd.md

```diff
@@ -0,0 +1,303 @@
+---
+id: gpu-data-processing-inside-lxd
+summary: Accelerate data processing within LXD containers by enabling direct access to your Nvidia GPU's CUDA engine. 
+categories: lxd
+tags: lxd, cuda, nvidia, gpu, big data
+difficulty: 4
+published: 2017-07-12
+author: Canonical Web Team <webteam@canonical.com>
+
+---
+
+# GPU data processing inside LXD
+
+## Overview
+Duration: 3:00
+
+Hugely parallelised GPU data processing, using either [CUDA][nvidiacuda] or [OpenCL][opencl], is changing the shape of data science. It even has its own snappy acronym - GPGPU - General-purpose computing on graphics processing units.  It's no surprise, then, that flexible, scalable access to these GPU resources is becoming a key requirement in many cloud deployments (see the Canonical Distribution of [Kubernetes][kubernetes] for a good example). But it's not always easy, nor cheap, to get started. Unless you use [LXD][lxd].
+
+Thanks to LXD's unrivalled density in real-world cloud deployments, and its ability to run locally, LXD provides one of the best ways of experimenting with cloud-like GPU data processing. It enables you to create local scalable deployments using nothing more than a PC with a GPU or two. As we'll now demonstrate.
```

Comment:
too many LXDs in first line. Maybe  "... to run locally, makes it a game-changing tool for experimenting..."